config/locales/devise.es.yml

```diff
@@ -0,0 +1,63 @@
+es:
+  devise:
+    confirmations:
+      confirmed: "Tu correo electrónico ha sido confirmado exitosamente."
```

Comment:
@csprayregen Do you mind helping me review these translations? Feel free to edit and push changes to this branch.